.github/workflows/conda-nightly-cpu.yml

```diff
@@ -0,0 +1,36 @@
+name: Push conda nightly
```

Comment:
According my [previous comments](https://github.com/pytorch/serve/pull/1685#issuecomment-1160739682), this workflow shouldn't exist separately. conda cpu nightly binary&publish should be within pypi nightly cpu build workflow.